diff --git a/node_modules/jira-prepare-commit-msg/bin/config.js b/node_modules/jira-prepare-commit-msg/bin/config.js
index b51d947..b268a2a 100644
--- a/node_modules/jira-prepare-commit-msg/bin/config.js
+++ b/node_modules/jira-prepare-commit-msg/bin/config.js
@@ -33,6 +33,7 @@ async function loadConfig(configPath) {
                 '.jirapreparecommitmsgrc.yaml',
                 '.jirapreparecommitmsgrc.yml',
                 'jira-prepare-commit-msg.config.js',
+                'jira-prepare-commit-msg.config.cjs',
             ],
         });
         const config = configPath ? await explorer.load(resolveConfig(configPath)) : await explorer.search();
